### PR TITLE
GFORMS-877 Upgrade to Play 2.6

### PR DESCRIPTION
**Pull request by girishkamat**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -131,22 +105,15 @@ feature {
 }
 
 basicAuth {
-  users = ""
+  users = "girish.kamath@digital.hmrc.gov.uk"
```
Comment (josef-vlach) [inline conf/application.conf:108]: This should be removed.

File: conf/prod.routes [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Add all the application routes to the app.routes file
-->         /eeitt-admin-frontend              app.Routes
-->         /                          health.Routes
-->     	   /template                  template.Routes
-
-GET        /admin/metrics             com.kenshoo.play.metrics.MetricsController.metrics
+->         /submissions                  app.Routes
```
Comment (josef-vlach) [inline conf/prod.routes:2]: `/submission` is dedicated to gforms, it should not be used.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+$govuk-assets-path: "/submissions/assets/lib/govuk-frontend/govuk/assets/";
+$hmrc-assets-path: "/submissions/assets/lib/hmrc-frontend/hmrc/";
```
Comment (josef-vlach) [inline app/assets/stylesheets/application.scss:2]: `submissions` is gform context. `eeitt-admin-frontend` should be used instead.

**Review by josef-vlach — CHANGES_REQUESTED** [review]
Also javascript on Analytics is broken:

```
Uncaught ReferenceError: $ is not defined
```

jQuery seems to be required.

File: conf/prod.routes (hunk 2 above)
Comment (girishkamat) [reply conf/prod.routes:2]: sure, will update

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (girishkamat) [reply app/assets/stylesheets/application.scss:2]: sure, will update

File: conf/application.conf (hunk 1 above)
Comment (girishkamat) [reply conf/application.conf:108]: will remove, thanks

**Comment by girishkamat** [comment]
> Also javascript on Analytics is broken:
> 
> ```
> Uncaught ReferenceError: $ is not defined
> ```
> 
> jQuery seems to be required.

Good spot, just added jquery to the mix. its working now

**Pull request merged**